src/main/java/io/goobox/sync/common/overlay/OverlayHelper.java

```diff
@@ -118,6 +121,11 @@ private void init() {
                     logger.debug("Register {} with ID {} ({})", icon, String.valueOf(state.id()), state);
                     fileIconControl.registerIconWithId(
                             icon.toAbsolutePath().toString(), state.name(), String.valueOf(state.id()));
+                    try {
+                        Thread.sleep(1000);
```

Comment:
Let's have a comment why this 1 seconds sleep is required. Just to avoid removing it by accident in future.